analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -100,6 +100,8 @@ class Bower(
             Package(
```

Comment:
Commit message: "package maanger" -> "package manager".
In the last sentence, I would use plural, i.e. "The package managers can be..."

Reply:
Also, please use American English: s/Initialise/Initialize/